src/test/java/spoon/test/template/testclasses/EnumAccessTemplate.java

```diff
@@ -13,15 +12,15 @@ public void method() throws Throwable {
 	}
 	
 	@Parameter("_AnEnum_")
-	CtTypeReference<?> anEnum;
+	String __AnEnum_;

```

Comment:
ok, in fact I did not used it exactly like in example above, because it is probably not compilable, but I still argue that new contract between field name and parameter vale is not good because:
* it makes the refactoring of templates harder, because it is not enough to rename parameter value, but one must rename field and all field references too.
* The class name in parameter value and field name might represent different meaning and then it is jot possible to use selfexplanatory field name, but one must use field name which is based on class name.